flow/Knobs.cpp

```diff
@@ -345,3 +346,8 @@ void Knobs::trace() {
 	for(auto &k : bool_knobs)
 		TraceEvent("Knob").detail("Name", k.first.c_str()).detail("Value", *k.second);
 }
+
+void createGlobalFlowKnobs() {
```

Comment:
Either this is not needed or you should remove the global static initialization above. Currently it seems you create all knobs twice.